domains/f.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "iHategithub9000",
+           "email": "",
+           "discord": "836012649959522384",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.X5ydqojosseoZvJ5KWgT9xOHlThoEA_SUhcCqiMU_8PdrYbjy7Nw_45skTB9WM_-I5-jytdNY42ZMFO8jKGKzZeU8k6v-prPPEpFI3-frlVTFodnZb03puDilPvWV08Ms-3Gook9SJVHaibuFO3huscmzHR8LqgQWAdOOTlyIML-sxissxTom_3dXjHQEmtjkQrAOuTeBw6oZK7_V9uue3tpNyq3-FNCCKLLc--OskvgwVZB8CmqJUADUpgaxXz2IIvEezOiZyP_f0hII8MWR6VttQqtGiRoQKj5SvtoQsBHfhRfocNRY_RUOu1a6l30mx3TB04OLC-QSN-HfvOIQg.kmjyio3vkycaU6hX0FYzpg.PzGJ-TiHcN-8DiG6UvmqXxmXuPe0rVbIg6ykm5nXp5_jUsNVCHDKslCfUQp3D0Jlc8jY_ez7hvxsgtfOinJYu2oaI3qWs6nxQg-i8KJS-hI.tAjmi5HHDLJcfGDBSUEong"
+        },
+    
+        "record": {
+            "CNAME": "painscreen.is-a.dev"
```

Comment:
I don't think this will work. Instead I'd recommend using either URL records (unstable) or use [redirect.pizza](https://redirect.pizza) which is more stable.